vimmo/db/run_updates.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Find the directory where this script is located --> $0 is script being run, realpath gets the absolute path
+script_dir=$(dirname "$(realpath "$0")")
+
+# Find the root of the Git repository by tracing up from the script's directory
+# git -C "$script_dir" executes the git command as if it is in the script directory
+# rev-parse --show-toplevel returns the absolute path to the root of the Git repository
+# 2>dev/null redirects stderr to /dev/null, silencing errors
+repo_root=$(git -C "$script_dir" rev-parse --show-toplevel 2>/dev/null)
```

Comment:
Ideally shouldn't rely on git as we, if someone creates a git in a subfolder and they run the script from run folder, might cause an issue but should be fine for now and if we make sure we dont create other git subfolders. But probably a very niche scenario

Reply:
As far as I'm aware, `git -C` makes the commands run from the directory you provide (`$script_dir`), so it should't matter unless somebody created a .git in the specific directory where the script is. Could be wrong though